gravitee-am-ui/src/app/domain/applications/application/advanced/resources/resources.component.html

```diff
@@ -0,0 +1,67 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div fxLayout="column">
+  <div class="gv-form-section">
+    <div class="gv-form-section-title">
+      <h5>Resources</h5>
+      <mat-divider></mat-divider>
+    </div>
+    <div>
+      <p>Administrators of this application can list resources registered by resource owners via the UMA 2.0 Resource Set Registration Endpoint.</p>
+      <p>The endpoint defined by this specification at which the resource server registers resource sets it wants the authorization server to know about to apply access policies.</p>
```

Comment:
```suggestion
      <p><i>This endpoint defined by the UMA 2.0 specification is used by the resource server to register resource sets it wants the authorization server to protect and manage access policies.</i></p>
```
Suggestion pour compréhension?

Reply:
FP